spec/datadog/tracing/contrib/grpc/datadog_interceptor/server_spec.rb

```diff
@@ -70,11 +72,12 @@
     end
 
     context 'with an error' do
-      subject(:request_response) do
+      let(:request_response) do
```

Comment:
Is there any specific reason to change `subject` to `let` ?

Reply:
It's actually more consistent now with the other example blocks in these files, but mostly because the original subject had to be redefined on line 96.

Happy to swap this back if you prefer though.

Reply:
👋 @rfwroo , Thanks for the clarification. I think its fine and not a blocker at all. I have triggered our workflow. I believe it is ready to be merged after all the checks are successful. 

Really appreciate the great work! 👍